docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,740 @@
+
+.. raw:: html
+
+    <style type="text/css">
+    .center_vid {   margin-left: auto;
+                    margin-right: auto;
+                    display: block;
+                    width: 75%; 
+                }
+    </style>
+    <video class='center_vid' controls><source src="../_static/control_center_pics/configuration_screen_cap.mp4" type="video/mp4"></video>
+
+.. _config_values:
+
+Configuration Values
+========================
+Values currently set on the target module are displayed in the box on the right hand side of each parameter.
+
+.. image:: ../_static/control_center_pics/config_options.png
+
+To learn more about a parameter, click on the “i” icon on the right hand side
+
+.. image:: ../_static/control_center_pics/motor_dir_i_highlight.png
+
+Saving and Refreshing
+--------------------------
+With a spin box, you can either type in a new value or use the up and down arrows to change the value. With a dropdown menu, 
+simply click the box to list the options available, and click on the value you would like to set.
+
+Values that have been changed but not saved are indicated by a * icon. For example, if you change *Mode* from 
+Voltage to Velocity without saving or refreshing the value, we see
+
+.. image:: ../_static/control_center_pics/mode_changed.png
+
+From this point, there are two choices: Save or Refresh. Saving sets and saves the new value on the targeted module. Refreshing rereads the target 
+module's value, and sets it back into either the spin box or dropdown.
+
+Saving is indicated by the down arrow on the right hand side
+
+.. image:: ../_static/control_center_pics/save_button.png
+
+Refresh is indicated by the circular arrow
+
+.. image:: ../_static/control_center_pics/refresh.png
+
+In this example, when we click the refresh button, we see Velocity return to Voltage, and the star disappears.
+
+If we save Velocity, however, you will see the star disappear, and a message stating “Value Saved Successfully” appears in the Messages section.
+
+Configuration Tabs
+===================
+The Control Center provides 3 tabs of configuration options: General, Tuning, and Advanced. These are found on the left side of the GUI, 
+and the selected tab is indicated in blue.
+
+.. image:: ../_static/control_center_pics/config_tabs.png
+
+.. note::
+    The configuration parameters presented for each module family and firmware style vary. Not all parameters shown in this example will be available for all modules and firmware styles.
+
+General
+-----------
+The general tab provides access to some more basic module configuration parameters. Parameters here help configure attributes necessary for properly controlling 
+your module with your desired behaviors.
+
+For example, the general tab is where you will find the :ref:`Motor Direction parameter <throttle_direction>`.
```

Comment:
You'll only find it there for speed firmware, worth specifying that if this is meant to be general cross-firmware directions